project/src/store/reducer.ts

```diff
@@ -1,20 +1,53 @@
 import { createReducer } from '@reduxjs/toolkit';
-import { changeCity, markOfferOnCard, findOfferById, findOfferNearby, findOfferReviews } from './action';
+import { changeCity, markOfferOnCard, findOfferById, findOfferNearby, findOfferReviews, changeSortType } from './action';
 import { mockOffersList } from 'mocks/offers';
 import { mockOfferToShow } from 'mocks/offerToShow';
 import { mockOffersNearby } from 'mocks/offersNearby';
 import { mockReviewList } from 'mocks/review';
 import { InitialState } from '@customTypes/store';
+import { Offer } from '@customTypes/index';
 
 const initialState: InitialState = {
   city: 'Paris',
+  sortType: 'Popular',
   offers: mockOffersList,
   offerToShow: mockOffersList[-1],
   offersNearby: mockOffersList,
   offerReviews: mockReviewList,
   hoveredOffer: mockOffersList[-1],
 };
 
+function priceHighToLow(a: Offer, b: Offer) {
+  if (a.price > b.price) {
+    return -1;
+  }
+  if (a.price < b.price) {
+    return 1;
+  }
+  return 0;
+}
```

Comment:
```suggestion
function priceHighToLow(a: Offer, b: Offer) {
 return Math.sign(a.price - b.price);
}
```

Если там числа, можно так